### DISCUSSION TITLE
Visually differentiating character in dialogue

### DISCUSSION THREAD

**Discussion by danfriedz** [nathanhoad/godot_dialogue_manager#423] [Q&A]
https://github.com/nathanhoad/godot_dialogue_manager/assets/61813510/31c3547e-9c49-4198-a9e1-160224b83890

Hi,
In my game, I have two characters sitting next to each other. Is it possible to differentiate between the two characters in dialogue?
Possible examples might be.

- Colour coding specific to each character. This would involve changing text color mid-dialogue. Does it support something like [colour=RED]some dialogue text[\colour=RED]?
- Changing the text alignment mid-dialogue ( so the character on the right-hand side is right justified)

I could probably find workarounds for these, changing the theme via variables within the script perhaps? but I wanted to see if there is any built-in functionality for this currently or planned.

**Comment by nathanhoad** [comment]
You could do it with the BBCode tags but a better way would be to handle that in your custom balloon with theme/colour overrides on the Dialogue Label or character name label.
